corehq/apps/cloudcare/static/cloudcare/js/form_entry/entrycontrols_full.js

```diff
@@ -373,6 +373,9 @@ function ComboboxEntry(question, options) {
                 matcher: function() {
                     return $input.val();
                 },
+                sorter: function(query, data) {
+                    return data;
+                },
```

Comment:
this needs to be changed. the default sorter would eliminate items that didn't directly match the query... which is dumb considering there's a function to filter the items already.